internal/api/courses/users/list.go

```diff
@@ -0,0 +1,29 @@
+package users
+
+import (
+	"github.com/edulinq/autograder/internal/api/core"
+	"github.com/edulinq/autograder/internal/model"
+)
+
+type ListRequest struct {
+	core.APIRequestCourseUserContext
+	core.MinCourseRoleGrader
+	Users core.CourseUsers `json:"-"`
+}
+
+type ListResponse struct {
+	Users []*core.CourseUserInfo `json:"users"`
+}
+
+func HandleList(request *ListRequest) (*ListResponse, *core.APIError) {
```

Comment:
One of the return types is an APIError but the only error that gets returned is nil. Is this on purpose?

Reply:
All Handler's return a core.APIError, so it is here to maintain consistency.

Reply:
Gotcha ok you can resolve this then.